src/Banking/Operation.php

```diff
@@ -55,6 +55,9 @@ class Operation
     /** @var OperationDetail|null */
     private $details;
 
+    /** @var OperationDetail[] */
+    private $subDetails = [];
```

Comment:
I think your implementation is better than mine.
We should change the `$details` property to be an array, keep the `getDetails` method for BC reason, and just add an method like `getAllDetails` instead of adding `$subDetails` property.

WDYT?

Reply:
ok so if we dont want BC `getDetails` will return the first element :)

Reply:
Yes!

Reply:
done